timescaledb/getting-started/query-data.md

```diff
@@ -1,153 +1,136 @@
-# 5. Query your data
+# Query your data
 
 With TimescaleDB, there's no need to learn a custom query language. **TimescaleDB
-supports full SQL**. This means you can put your SQL knowledge to good use and
-use the rich ecosystem of PostgreSQL tools you know and love.
-
-For example, here's how to find the average temperature for each city in the past 2 years:
-
-```sql
---------------------------------
--- Average temperature per city
--- in past 2 years
---------------------------------
-SELECT city_name, avg(temp_c)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '2 years'
-GROUP BY city_name;
-```
-
-And here's how to find the total snowfall for each city in the past 5 years:
-
-```sql
---------------------------------
--- Total snowfall per city
--- in past 5 years
---------------------------------
-SELECT city_name, sum(snow_1h_mm)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '5 years'
-GROUP BY city_name;
-```
-
-
-<highlight type="tip">
-Fun fact: TimescaleDB adds important enhancements to the PostgreSQL query planner
-that improve query reusability for INTERVAL predicates, something PostgreSQL does
-not have.
-</highlight>
-
-## Advanced SQL functions for time-series data
+supports full SQL**. You can use your SQL knowledge along with the rich 
+ecosystem of PostgreSQL tools, and add the extra features and performance of
+TimescaleDB.
+
+
+Here are some query examples so you can get familiar with using SQL alongside
+popular TimescaleDB functions.
+
+## Basic SQL queries
+### Select all stock data from the last day.
+
+   This query uses the [`WHERE`][clause-expressions] clause to only show results for
+   timestamps within the last day. 
+
+   ```sql
+   SELECT * FROM stocks_real_time srt
+   WHERE time > now() - INTERVAL '1 day';
```

Comment:
```suggestion
   WHERE time > now() - INTERVAL '4 days';
```

Reply:
I'm not sure about this, but if the user runs the original query on Monday, nothing will show up probably because trading is closed on weekends. 4 days as the interval seems more safe

Reply:
Yes, this happened to me on a Monday. Maybe a note about the time interval at the beginning of the section can help.  In my case, even an interval of 4 and 5 days did not work.
tsdb=# SELECT * FROM stocks_real_time srt                 
WHERE time > now() - INTERVAL '4 day';
 time | symbol | price | day_volume 
------+--------+-------+------------
(0 rows)

tsdb=# SELECT * FROM stocks_real_time srt
WHERE time > now() - INTERVAL '6 day';
tsdb=# SELECT * FROM stocks_real_time srt
WHERE time > now() - INTERVAL '5 day';
 time | symbol | price | day_volume 
------+--------+-------+------------
(0 rows)